frontend/app/src/components/home/Home.svelte

```diff
@@ -277,6 +278,20 @@
         return found;
     }
 
+    function redirectToFirstChat(scope: ChatListScope): boolean {
+        if (scope.kind === "community") {
+            return redirectToFirstChannel(scope.id);
+        } else if (scope.kind === "favourite" && scope.communityId !== undefined) {
```

Comment:
Favourite + community?

Reply:
TBH I'm not sure what that means. I have simplified in any case...